### PR TITLE
chore(deps): Move peerDependencies

### PR DESCRIPTION
**Pull request by talend-glorieux**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our [guidelines](https://github.com/Talend/react-talend-forms/blob/master/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

react-jsonschema-form isn't defined as a dependency.
rjsf-jsonschema-form is part of the peerDependencies.

**What is the new behavior?**

react-jsonschema-form and rjsf-jsonschema-form are now dependencies instead of peerDependencies.

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,7 +48,8 @@
     "react": "^15.3.1",
     "react-bootstrap": "^0.30.3",
     "react-dom": "^15.3.1",
-    "rjsf-material-design": "^1.0.1"
+    "rjsf-material-design": "^1.0.1",
+    "react-jsonschema-form": "^0.41.1"
```
Comment (frassinier) [inline package.json]: Which use case needs to deal with react-jsonschema-form version? :/

Comment (talend-glorieux) [inline package.json]: For now we've always used the peer dependencies on our libraries. It forces the user to handles the version conflicts by himself.
react-jsonschema-form is simply one of those peerDependencies.

File: package.json [hunk 2]
```diff
@@ -44,11 +44,14 @@
     "semantic-release": "^4.3.5",
     "webpack": "^1.13.1"
   },
+  "dependencies": {
+    "rjsf-material-design": "^1.0.1",
+    "react-jsonschema-form": "^0.41.1"
+  },
```
Comment (jmfrancois) [inline package.json]: we just have to be sure that no one will include this deps.

That LGTM

Comment (talend-glorieux) [inline package.json]: It's essentially a way for us to make sure that people don't use those library directly.

**Pull request merged**